core/variant/variant_utility.cpp

```diff
@@ -1827,6 +1751,8 @@ void Variant::_register_variant_utility_functions() {
 	FUNCBINDR(rid_from_int64, sarray("base"), Variant::UTILITY_FUNC_TYPE_GENERAL);
 
 	FUNCBINDR(is_same, sarray("a", "b"), Variant::UTILITY_FUNC_TYPE_GENERAL);
+
+	FUNCBINDVARARGS(join, sarray(), Variant::UTILITY_FUNC_TYPE_GENERAL);
```

Comment:
I don't think this new bind is needed, it's just a simpler version of `"".join([ a, b, c ])`, only difference is that it's a bit faster